tensorflow/contrib/cmake/tf_core_kernels.cmake

```diff
@@ -92,6 +92,8 @@ if(tensorflow_BUILD_CONTRIB_KERNELS)
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_manager.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_ops.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/ops/nccl_ops.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/core/kernels/periodic_resample_op.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/core/ops/array_ops.cc"
```

Comment:
I think there's a typo in the filenames here (there's no `core/` subdirectory in the added files). Can you try the following instead?

```
      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/kernels/periodic_resample_op.cc"
      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/ops/array_ops.cc"
```